src/language/parser.js

```diff
@@ -943,6 +943,9 @@ function parseUnionTypeDefinition(lexer: Lexer<*>): UnionTypeDefinitionNode {
  */
 function parseUnionMembers(lexer: Lexer<*>): Array<NamedTypeNode> {
```

Comment:
Every method in the parser has parse language atop it. It would be nice to include this here as well